### ISSUE TITLE
resume from checkpoint

### ISSUE THREAD

**Issue by bmaltais** [jcjohnson/fast-neural-style#24]
Thank you for implementing the resume from checkpoint option.

I noticed that using it was not resuming with proper loss. It was much higher than when I had stopped the training... but playing with the learning I noticed the following:

Lowering the learning rate to 1e-4 instead of the default 1e-3 will actually properly resume training without raising the loss.

Not sure if there is a science to this but worth trying if you restart training.

**Comment by rayset** [comment]
I believe the learning rate decays over time, so when you restart you should set it to a value lower than the starting one.

I haven't checked if the code is supposed to do it on its own tho.

**Comment by piteight** [comment]
Hi, I have noticed that my net is not making any progress. I use this code to do it from checkpoint:
h train.lua -resume_from_checkpoint checkpoint2.t7 -h5_file ../Baz.h5 -style_image ../palemka.png -style_image_size 256 -content_weights 1.0 -style_weights 5.0 -gpu 0 -use_cudnn 1 -batch_size 2

It was trained 3 times, and the output is almost the same as it was firstly.
every time I train it starts from epoch 0.000000, Iteration 0 and goes to 40000.
